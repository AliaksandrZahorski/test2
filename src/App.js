import React, {Component} from 'react';
import MyButton from './MyButton'
import InputForm from './InputForm';
import CollapseDemo from './CollapseDemo';
import Table from './Table';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import FlatButtonExampleSimple from './FlatButtonExampleSimple';

import { header, data, locale } from './fixtures';

class App extends Component {
  render() {
    return (
      <div>
      <MuiThemeProvider>
        <FlatButtonExampleSimple />
      </MuiThemeProvider>
        {/*
        <MyButton state={false}>
          testChild
        </MyButton>
        <InputForm inputValue={1}/>
        <CollapseDemo />
        <Table header={header} data={data} locale={locale}/>
        */}
      </div>
    );
  }
}

export default App
