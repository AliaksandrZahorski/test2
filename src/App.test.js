import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

jest.mock('./MyButton', () =>  'MyButton' )
jest.mock('./InputForm', () =>  'InputForm' )

describe('App', () => {
  it('renders correctly', () => {
    const app = renderer.create(
      <App />
    ).toJSON()
    expect(app).toMatchSnapshot()
  })
})
