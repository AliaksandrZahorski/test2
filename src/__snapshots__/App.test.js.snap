// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <MyButton
    state={false}
  />
  <InputForm
    inputValue={1}
  />
</div>
`;
